design/attribute.go

```diff
@@ -511,3 +511,36 @@ func (v *ValidationExpr) Dup() *ValidationExpr {
 		Required:  req,
 	}
 }
+
+// IsSupportedValidationFormat checks if the validation format is supported by goa.
+func (a *AttributeExpr) IsSupportedValidationFormat(vf ValidationFormat) bool {
```

Comment:
This test seems a bit odd - not sure we need to protect against user force casting to `ValidationFormat`? Maybe a better way to approach this is to make the `ValidationFormat` type private - this way DSL writers would be forced to use the public values defined for the enum.